addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -356,10 +356,11 @@ registerModel({
          */
         publicPartners: many2many('mail.partner'),
         /**
-         * Threads for which the current partner has a pending invitation
+         * Threads for which the current partner has a pending invitation.
+         * It is computed from the inverse relation for performance reasons.
          */
-        ringingThreads: many2many('mail.thread', {
-            compute: '_computeRingingThreads',
```

Comment:
@seb-odoo forgot to remove obsolete compute method

Reply:
It was done in the original PR, I guess mistake at forward-port, I'll fix it.